src/components/items/Event.jsx

```diff
@@ -119,7 +119,7 @@ function Event(props) {
         <Nav tabs>
           <NavItem>
             <NavLink
-              className={{ active: activeTab === "1" }}
+              /* className={{ active: activeTab === "1" }} */
```

Comment:
Unsure of why this is commented out.

Reply:
These are invalid and were generating an error on the console.log className has to be a string.